BoardConfigCommon.mk

```diff
@@ -60,7 +62,6 @@ BOARD_KERNEL_CMDLINE := \
     msm_rtb.filter=0x3F \
     ehci-hcd.park=3 \
     androidboot.bootdevice=msm_sdcc.1
-BOARD_MKBOOTIMG_ARGS := --ramdisk_offset 0x02000000
```

Comment:
Read the note above. This flag is included in the mkbootimg.mk. You will have to merge the boot logo into Lumia 720 device tree in order to make this working.